pkg/kubectl/cmd/util/BUILD

```diff
@@ -16,6 +16,9 @@ go_library(
         "factory.go",
         "helpers.go",
         "printing.go",
+        "ring0_factory.go",
```

Comment:
I don't like the ring names.  It's an arbitrary abstraction that has nothing to do with the actual use of the code.  But the approach is fine.  Call it "core" or "client" or "printing".